docker-hub/builds/index.md

```diff
@@ -302,16 +302,9 @@ You could also use capture groups to build and label images that come from vario
 
 ### Build images with BuildKit
 
-You can enable the BuildKit builder by setting the `DOCKER_BUILDKIT=1`
-[environment variable](#environment-variables-for-builds) in the
-[Configure automated build settings](#configure-automated-build-settings) section.
-Refer to the [build images with BuildKit](../../develop/develop-images/build_enhancements.md)
+Autobuilds use the buildKit build-system by default. If you want to turn it off, add the [environment variable](index.md#environment-variables-for-builds){: target="_blank" rel="noopener" class="_"} DOCKER_BUILDKIT=0. Refer to the [build images with BuildKit](../../develop/develop-images/build_enhancements.md)
```

Comment:
Nits: Can we remove the hyphen in `build-system` -> `build system` please?
Also to capitalise BuildKit.

I also prefer the `If you want to use the legacy Docker build system` rather than `If you want to turn it off` line. Or we could do something like `If you want to turn it off ...  [ ] ... This will then use the legacy docker build system.`

Reply:
Hey! @mikeparker Sure, I updated it with the legacy build system